File: kotlin-ide/j2k/new/testData/newJ2k/tryWithResource/MultipleResources.java
//file
import java.io.*;

public class C {
    void foo() throws IOException {
        try(ByteArrayInputStream input = new ByteArrayInputStream(new byte[10]);
            OutputStream output = new ByteArrayOutputStream()) {
            output.write(input.read());
            output.write(0);
        }
    }
}

File: nj2k/testData/newJ2k/tryWithResource/MultipleResources.kt
import java.io.ByteArrayInputStream
import java.io.ByteArrayOutputStream
import java.io.IOException

class C {
    @Throws(IOException::class)
    internal fun foo() {
        ByteArrayInputStream(ByteArray(10)).use { input ->
            ByteArrayOutputStream().use { output ->
                output.write(input.read())
                output.write(0)
            }
        }
    }
}
